docs/containers-registries.conf.5.md

```diff
@@ -244,6 +250,8 @@ Given the above, a pull of `example.com/foo/image:latest` will try:
 
 in order, and use the first one that exists.
 
+Note that a mirror is associated only with the previous registry.  Pulling `registry.com/image:latest` will hence only reach out to `mirror.registry.com` and not use the one associated to the other registry.
```

Comment:
Would it be clearer (or much less clear??) to explicitly list the [`"example-mirror-0.local"`, `"example-mirror-1.local"`] names instead of saying “the other registry”? Or maybe both, “and use neither `"example-mirror-0.local"` nor `"example-mirror-1.local"`, which only apply to the `example.com/foo` namespace”— OTOH that seems _way_ too long.

Reply:
Good idea!  We can keep it shorter by saying "the mirror of `example.com/foo`"